README.md

```diff
@@ -352,6 +340,92 @@ Returns one of the following in order of preference:
 In order for this to be most useful, you must configure your controller
 as described in [Change Journaling](#change-journaling) above.
 
+### Testing
+
+If you use RSpec, you can test for journaling behaviors with the
+`journal_event(s)` and `journal_changes_to` matchers. First, make sure to
+require `journaled/rspec` in your spec setup (e.g. `spec/rails_helper.rb`):
+
+```ruby
+require 'journaled/rspec'
+```
+
+#### Checking for specific events
+
+The `journal_event` and `journal_events` matchers allow you to check for one or
+more matching event being journaled:
+
+```ruby
+expect { my_code }.to journal_event(name: 'foo')
+expect { my_code }.to journal_events({ name: 'foo', value: 1 }, { name: 'foo', value: 2 })
```

Comment:
> should we make it clear in the readme whether the singular (or plural) forms will fail to match if additional journal entries are logged? Should there be an option to make it work the other way if requested?

cc @jmileham 

Good question. In my experience trying this out locally within a large codebase, additional (and mostly uninteresting-to-the-test) events _are_ logged more often than not, and so I wanted to avoid forcing implementing devs to specify values or counts for extra, unrelated events. Particularly, I've found that adding new events can cause a lot of unrelated tests to fail because they were only expecting certain events and now there's one more in the mix.

What I think _is_ very important is that for every positive assertion, there is a negative assertion somewhere nearby. But I couldn't think of a good way to encourage that via the DSL, other than by providing easy access to the negated matchers.

Another thing we could do here is provide chainable counters -- like `.once` and `.times(3)` -- or simply default to ensuring that each expected event is only matched against once. I don't think I'm explicitly asserting that right now, so it would be an extra bit of testing and would probably result in something like `expected: {...} to be emitted once, but was emitted more than once: {...}, {...}`. Does that make sense?

Reply:
I agree with the behavior ignore-extra-events behavior.  We should call it out in the readme. Also fine with not offering a "these and only these events will be emitted" variant for now.

Having an implementation like [receive counts](https://relishapp.com/rspec/rspec-mocks/docs/setting-constraints/receive-counts) would be cool, defaulting to once like receive does. If you think it's worth it.

Reply:
To summarize the asks:

* Lets call out the fact that it doesn't care about extra events in the readme

Optionally add the receive counts behavior if you wanna